js/apps/admin-ui/src/user/utils.ts

```diff
@@ -25,3 +25,6 @@ export const fieldName = (attribute: UserProfileAttributeMetadata) =>
   isRootAttribute(attribute.name)
     ? attribute.name
     : `attributes.${attribute.name}`;
+
+export const isLightweightUser = (userId?: String) =>
```

Comment:
For primitive values in TypeScripts (e.g. `number`, `string`, etc), we use the lowercase variant.
```suggestion
export const isLightweightUser = (userId?: string) =>
```